Obsidian.API/Containers/Container.cs

```diff
@@ -30,27 +30,12 @@ public override int AddItem(ItemStack item)
 
         if (this.IsPlayerInventory)
         {
-            for (int i = 45; i > 8; i--)
-            {
-                var invItem = this.items[i];
-
-                if (invItem?.Type == item.Type)
-                {
-                    if (invItem.Count >= 64)
-                        continue;
-
-                    invItem.Count += item.Count;
-
-                    return i;
-                }
-
-                if (invItem == null)
-                {
-                    this.items[i] = item;
+            int player_hotbar_start = 36, player_hotbar_end = player_hotbar_start + 8;
+            int player_main_inventory_start = 9, player_main_inventory_end = player_hotbar_start - 1;
```

Comment:
Generally constant values are defined in the beginning of a class, in all caps. e.g. const int PLAYER_HOTBAR_START, PLAYER_HOTBAR_END

Reply:
that aside looks great

Reply:
In C#, constants are **always** in PascalCase.

Reply:
oops. yet still they don't belong inline

Reply:
> In C#, constants are **always** in PascalCase.

I'll change it.

Reply:
Changed it.